spec/views/users/users_show_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+require 'capybara/rspec'
+
+base_url = 'http://localhost:3000'
+
```

Comment:
- [ ] If we go by the content of this file, I think it the file name should be `users_index_spec.rb` not `users_show_spec.rb`. 👍 
- [ ] Please kindly add the missing test as well. ℹ️ 

<img width="402" alt="Untitled" src="https://github.com/Naledi-Dikgale/blog-app/assets/103386183/38ab17e0-9f2d-47f3-8968-1b0d66d5bee5">